aiida_fleur/workflows/eos.py

```diff
@@ -69,19 +64,13 @@ def define(cls, spec):
         spec.input("wf_parameters", valid_type=Dict, required=False)
         spec.input("structure", valid_type=StructureData, required=True)
 
-        spec.outline(
-            cls.start,
-            cls.structures,
-            cls.converge_scf,
-            cls.return_results
-        )
+        spec.outline(cls.start, cls.structures, cls.converge_scf, cls.return_results)
```

Comment:
In some cases the automatic formatting broke how the outline looks.
I think this should be fixed for all the workchains.